subprojects/packagefiles/mpdecimal/meson.build

```diff
@@ -194,9 +194,9 @@ mpd_header_config_fullcov = '''
 machine = get_option('machine')
 if machine == 'auto'
   if cc.sizeof('size_t') == 8
-    if conf.get('HAVE_GCC_ASM_FOR_X64') == 1
+    if conf.get('HAVE_GCC_ASM_FOR_X64', 0) == 1
```

Comment:
None of these `conf.get()`s are actually used by the software. They match the configure.ac from upstream, which defines them as `AC_DEFINE` but never use them... they are only checked later in configure.ac to set the machine as we do here.

Reply:
I think 0ca96efc582e55711fdb23edf0412b0a411a4932 is a better general fix for this.